cmd/src/debug_kube.go

```diff
@@ -0,0 +1,320 @@
+package main
+
+import (
+	"archive/zip"
+	"bytes"
+	"context"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+
+	"golang.org/x/sync/errgroup"
+
+	"golang.org/x/sync/semaphore"
+
+	"github.com/sourcegraph/sourcegraph/lib/errors"
+	"github.com/sourcegraph/src-cli/internal/exec"
+)
+
+func init() {
+	usage := `
+'src debug kube' invokes kubectl diagnostic commands targeting kubectl's current-context, writing returns to an archive.
+Usage:
```

Comment:
This needs a newline between the slug and usage section:

```suggestion
'src debug kube' invokes kubectl diagnostic commands targeting kubectl's current-context, writing returns to an archive.

Usage:
```